applications/models.py

```diff
@@ -239,6 +239,21 @@ def set_status(self, status, user=None):
     def get_status(self):
         return self.statuses.last()
 
+    def get_application_events_by_basket(self):
+        handled_application_events: [ApplicationEvent] = {}
+        for basket in sorted(
+            self.application_round_baskets.all(), key=lambda k: k.order_number
```

Comment:
This is merely a taste issue; `self.application_round_baskets.order_by("order_number")` would be the django way 

obviously performance varies between ordering in memory and by the db but I think this case we have quite low data mass for baskets per application rounds so it does not matter.

Reply:
Yeah, that looks better.